lib/rspec/rails/view_path_builder.rb

```diff
@@ -0,0 +1,27 @@
+module RSpec
+  module Rails
+    # Builds paths for view specs using a particular route set.
+    class ViewPathBuilder
+      def initialize(route_set)
+        self.class.send(:include, route_set.url_helpers)
+      end
```

Comment:
Is there a benefit to using this over storing the `route_set` and calling `route_set.url_for` below?